### DISCUSSION TITLE
How to skip first minor release like X.Y.0 ?

### DISCUSSION THREAD

**Discussion by SystemZ** [renovatebot/renovate#32259] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### Please tell us more about your question or problem

Hi, I love renovate, it helped me a lot already to maintain my homelab 🚀

I'm stuck at fine tuning config.  
I would like to skip versions for a selected list of docker images, ending with patch 0 so like 1.1.0, 1.2.0 etc and allow 1.1.1, 1.2.1, 1.2.3 etc

**Comment by rarkins** [comment, marked as answer]
Use allowedVersions
